doc/getting-started/overview_dev.rst

```diff
@@ -81,18 +84,26 @@ that ACRN offers:
 
 ACRN offers three types of VMs:
 
-* **Pre-launched User VMs**: These VMs run independently of other VMs and own
-  dedicated hardware resources, such as a CPU core, memory, and I/O devices.
-  Other VMs may not even be aware of the existence of pre-launched VMs. The
-  configuration of these VMs is static and must be defined at build time. They
-  are well-suited for safety-critical applications.
-
-* **Service VM**: This VM is required for scenarios that have post-launched VMs.
-  It controls post-launched VMs and provides device sharing services to them.
-  ACRN supports one Service VM.
-
-* **Post-launched User VMs**: These VMs share hardware resources. Unlike
-  pre-launched VMs, you can change the configuration at run-time. They are
+* **Pre-launched User VMs**: These VMs are automatically launched at boot time
+  by the hypervisor. They run independently of other VMs and own dedicated
+  hardware resources, such as a CPU core, memory, and I/O devices. Other VMs,
+  including the Service VM, may not even be aware of a pre-launched VM’s
+  existence. The configuration of pre-launched VMs is static and must be defined
+  at build time. They are well-suited for safety-critical applications.
```

Comment:
Can we add a note along those lines: `and where very strict isolation, including from the Service VM is desirable`?